conf.d/splunk.yaml

```diff
@@ -0,0 +1,8 @@
+---
+init_config:
+  default_timeout: 30
+
+instances:
+  - url: https://localhost:8089
+    username: obsrobot
+    password: foobar
```

Comment:
Does this config actually work? Above,
```
+        username = self.init_config.get('username')
+        password = self.init_config.get('password')
```
... so I'm expecting to find username and password on `init_config`